openraft/tests/fixtures/mod.rs

```diff
@@ -146,6 +147,9 @@ where
     /// Nodes which are isolated can neither send nor receive frames.
     isolated_nodes: Arc<Mutex<HashSet<C::NodeId>>>,
 
+    /// emulate network unreachable
+    network_failed: Arc<AtomicBool>,
```

Comment:
Be specific about what it does: it is meant to emit an error when `connect()` is called. Thus this field could be better changed to be `emit_connection_error`, or `fail_connect`